tests/rigctl_parse.c

```diff
@@ -3009,6 +3030,10 @@ declare_proto_rig(get_parm)
             fprintf(fout, "%s\n", val.s);
             break;
 
+        case RIG_CONF_BINARY:
+            dump_hex((unsigned char *)buffer, val.b.l);
```

Comment:
Hi Malcolm.  This addition of dump_hex() to rigctl_parse.c is causing a linker error on my MinGW build:

```
make[1]: Entering directory '/home/nate/builds/hamlib-4.0~git/tests'
  CC       rigctl-rigctl.o
  CC       rigctl-rigctl_parse.o
  CC       rigctl-dumpcaps.o
  CC       rigctl-sprintflst.o
  CCLD     rigctl.exe
rigctl-rigctl_parse.o: In function `rigctl_get_parm':
/home/nate/builds/hamlib-4.0~git/tests/rigctl_parse.c:3049: undefined reference to `dump_hex'
collect2: error: ld returned 1 exit status
Makefile:949: recipe for target 'rigctl.exe' failed
make[1]: *** [rigctl.exe] Error 1
make[1]: Leaving directory '/home/nate/builds/hamlib-4.0~git/tests'
Makefile:605: recipe for target 'install-recursive' failed
```

It's not easy to find out if the `dump_hex` symbol is exported into the `libhamlib.dll.a` but my check for the string shows it to not be found while the individual `debug.o` file does show the string.  I know just enough about MinGW sometimes to get things working but often I am stumped.  Obviously `dump_hex` is available on Linux as rigctl links without issue, so this is just a MinGW issue at the moment.  I will note that `dump_hex` is used in the barret backend and in iofunc.c.

Mike, do you have an idea here?  If we can't resolve this, perhaps the `dump_hex` will need to be done another way.